ui/src/organizations/components/Tasks.tsx

```diff
@@ -10,10 +10,12 @@ import FilterList from 'src/shared/components/Filter'
 
 // Types
 import {Task} from 'src/api'
+import {deleteTask} from 'src/tasks/api/v2/index'
```

Comment:
same thing here, there's a `Tasks` component in src/tasks/components, perhaps we could make that component more generic so it can be used in both places?

Reply:
Creating a new issue for this. Thanks!